autometa/common/external/diamond.py

```diff
@@ -330,6 +307,8 @@ def blast(
         provided `blast_type` is not 'blastp' or 'blastx'
     OSError
         Diamond execution failed
+    subprocess.CalledProcessError
+        Failed to run blast
     """
     if not os.path.exists(fasta):
         raise FileNotFoundError(fasta)
```

Comment:
1. I think `if verbose` can be a handy tool if someone is running the script as a standalone module. What do you think about it? 
Else I can remove the `if verbose` statements and just use the logging statement for all the functions.

Reply:
I think we should keep `tmpdir` as an input from the user. It may be the case that the user needs to specify a different directory rather than the temporary directory like in the case of space issues. I don't think we should use a `tempdir` flag with 1diamond` if we are randomly creating a temporary directory, as `diamond` itself creates a temporary directory to use. 
From what I read from the diamond documentation the `tempdir` flag is only needed in cases where the output directory does not has enough space available.
Let me know what you think about it